python_modules/dagster/dagster/core/events/__init__.py

```diff
@@ -874,13 +875,15 @@ def resource_init_success(
     ) -> "DagsterEvent":
 
         metadata_entries = []
-        for resource_key in resource_instances.keys():
-            resource_obj = resource_instances[resource_key]
-            resource_time = resource_init_times[resource_key]
-            metadata_entries.append(
-                MetadataEntry.python_artifact(
-                    resource_obj.__class__, resource_key, "Initialized in {}".format(resource_time)
-                )
+        for key in resource_instances.keys():
+            metadata_entries.extend(
+                [
+                    MetadataEntry(
+                        key,
+                        value=MetadataValue.python_artifact(resource_instances[key].__class__),
+                    ),
+                    MetadataEntry(f"{key}:init_time_ms", value=resource_init_times[key]),
+                ]
```

Comment:
screenshot of what this looks like in dagit? just want to ensure reasonable UX

Reply:
<img width="1100" alt="image" src="https://user-images.githubusercontent.com/1531373/158903080-c908e4f0-2eb2-4e84-bacf-43ce7e895798.png">

Reply:
looks like we can drop `_ms` since its a datetime str value not a number like i thought